README.md

```diff
@@ -1,2 +1,55 @@
-# portfolio_website
-Portfolio: setup and mobile version skeleton
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio: setup and mobile version skeleton
+
+## Description
+> this is the first milestone in the process of creating my portfolio website 
+
+![screenshot](./mobile_header-shot.PNG)
+
+
+## Built With
+
+- html
+- css
+
+
+## Requirements
+- Make sure that there is no linter errors.
+- Make sure that the Gitflow is used correctly
+- Make sure that the work is well Documented
+
+## General Procedures
+- Documented the project in a professional way
+- Created a new GitHub repo
+- Created a new branch according to the Github flow rules
+- Added .gitignore file
+- I set up a linter for HTML and CSS and  made sure that any files that are not relevant are ignored by git.
+- Added descriptive README file to my project
+- build the first 3 sections of the mobile website using the template you chose
+- Opened a pull request.
```

Comment:
- [ ] Always remember to add information on how a user can get and run the project on their local machine.